### ISSUE TITLE
Low GPU utilization when "Optimize game latency" turn on

### ISSUE THREAD

**Issue by G00dog** [alvr-org/ALVR#2326]
When I enable the "Optimize game latency", my FPS drops, and GPU utilization decreases. 
When I disable "Optimize game latency," FPS is stable, and GPU utilization increases, but the latency becomes inconsistent, making it difficult to use.

The picture is the difference when looking at exactly the same place.
![optimize on](https://github.com/user-attachments/assets/f5232587-cad0-43e6-ac2f-0103db018f41) ![optimize off](https://github.com/user-attachments/assets/bad23d2c-91f5-45fa-a941-1a2eeff84260)
Maybe it's the same issue. [(https://github.com/alvr-org/ALVR/issues/1901)]

Is there a solution for this problem?

## General Troubleshooting
- [ ] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and successfully completed the setup wizard
- [ ] I read the [ALVR GitHub Wiki](https://github.com/alvr-org/ALVR/wiki)

### Hardware

**CPU** : Intel(R) Xeon(R) Gold 6238R CPU @ 2.20GHz

**GPU** : NVIDIA RTX A6000 X2

**GPU Driver Version** : 552.74

**Audio** :

### Installation
**ALVR Version** : 20.5

**ALVR Settings File** :

**SteamVR Version** : 2.74

**Install Type**:
- [ ] Packaged (`exe`, `deb`, `rpm`, etc)
- [ ] Portable (`zip`)
- [ ] Source

**OS Name and Version** (`winver` on Windows or `grep PRETTY_NAME /etc/os-release` on most Linux distributions):
Window 10

**Comment by zarik5** [comment]
Can you show the latency and fps graphs for both with and without "optimize game render latency"?

**Comment by G00dog** [comment]
![optimize on](https://github.com/user-attachments/assets/a09975b4-8c0c-4351-ba21-bdca2a52f46d)
![optimize off](https://github.com/user-attachments/assets/90b32b99-d6b5-4b99-858b-93532d78fd72)

Looks like a needle is turning on the optimization game render latency

**Comment by zarik5** [comment]
Yeah that bug seems to be caused by SteamVR not respecting the framerate we set. To be honest though, disabling that setting is wrong usage of the OpenVR driver API (the implementation details are quite complex). This works on Windows, but on Linux the implementation is completely different and that part of the code is skipped altogether.

**Comment by Vixea** [comment]
I think the issue is frame pacing here if you look at the graph even if the avg server fps is the clients I wouldn't be surprised if those small dips are noticeable?

**Comment by Vixea** [comment]
Anyway I don't get the lower util as being a bug its not its like complaining your browser isn't using 100% of your cpu

**Comment by G00dog** [comment]
<img width="961" alt="optimize on" src="https://github.com/user-attachments/assets/64b53286-9066-4d96-a60f-c32748012ab3">

<img width="959" alt="optimize off" src="https://github.com/user-attachments/assets/271c2222-e9f2-4b52-9f72-599ecfdc989e">


This image shows the difference in frame drops when 'Optimize Game Render Latency' is turned on versus turned off. 
I thought that if the GPU usage was as high with 'Optimize Game Render Latency' turned on as it is when it's turned off, it might result in better performance.
(Please understand that I’m not very knowledgeable about computers.)
